README.md

```diff
@@ -34,6 +34,11 @@ Onion links:
 
 [Alternative Invidious instances](https://github.com/omarroth/invidious/wiki/Invidious-Instances)
 
+## Screenshots
+<img src="/screenshots/invidious_popular.png?raw=true" height="140" width="280"> <img src="/screenshots/invidious_video.png?raw=true" height="140" width="280"> <img src="/screenshots/invidious_preferences.png?raw=true" height="140" width="280">
+
+<img src="/screenshots/invidious_popular_dark.png?raw=true" height="140" width="280"> <img src="/screenshots/invidious_video_dark.png?raw=true" height="140" width="280"> <img src="/screenshots/invidious_preferences_dark.png?raw=true" height="140" width="280">
+
```

Comment:
These need to be relative paths, looking at the [NewPipe README](https://github.com/TeamNewPipe/NewPipe/blob/dev/README.md) you can see there's also links to the images themselves, here's an example of what they might look like:

```markdown
[<img src="screenshots/invidious_popular.png?raw=true" height="140" width="280">](screenshots/invidious_popular.png?raw=true)
[<img src="screenshots/invidious_video.png?raw=true" height="140" width="280">](screenshots/invidious_video.png?raw=true)
[<img src="screenshots/invidious_preferences.png?raw=true" height="140" width="280">](screenshots/invidious_preferences.png?raw=true)
[<img src="screenshots/invidious_popular_dark.png?raw=true" height="140" width="280">](screenshots/invidious_popular_dark.png?raw=true)
[<img src="screenshots/invidious_video_dark.png?raw=true" height="140" width="280">](screenshots/invidious_video_dark.png?raw=true)
[<img src="screenshots/invidious_preferences_dark.png?raw=true" height="140" width="280">](screenshots/invidious_preferences_dark.png?raw=true)
```

Reply:
Well, they use markdown, I use HTML which is allowed in .md files. Anyway, I will change it.